jwst/datamodels/schemas/referencefile.schema.yaml

```diff
@@ -1,32 +1,203 @@
-allOf:
-- $ref: core.schema.yaml
-- type: object
-  properties:
-    meta:
-      type: object
-      properties:
-        reffile:
-          title: Information about the reference file
-          type: object
-          properties:
-            type:
-              title: Reference file type
-              type: string
-              fits_keyword: REFTYPE
-            pedigree:
-              title: The pedigree of the reference file
-              type: string
-              fits_keyword: PEDIGREE
-            description:
-              title: Description of the reference file
-              type: string
-              fits_keyword: DESCRIP
-            author:
-              title: Author of the reference file
-              type: string
-              fits_keyword: AUTHOR
-            useafter:
-              title: Use after date of the reference file
-              type: string
-              fits_keyword: USEAFTER
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      date:
+        allOf:
+        title: Date this file was created (UTC)
+        type: string
+        fits_keyword: DATE
+      origin:
+        title: Organization responsible for creating file
+        type: string
+        fits_keyword: ORIGIN
+      time_sys:
+        title: principal time system for time-related keywords
+        type: string
+        fits_keyword: TIMESYS
+      filename:
+        title: Name of the file
+        type: string
+        fits_keyword: FILENAME
+      filetype:
+        title: Type of data in the file
+        type: string
+        fits_keyword: FILETYPE
+      data_processing_software_version:
+        title: Data processing software version number
+        type: string
+        fits_keyword: SDP_VER
+      model_type:
+        title: Type of data model
+        type: string
+        fits_keyword: DATAMODL
+      telescope:
+        title: Telescope used to acquire the data
+        type: string
+        default: JWST
+        fits_keyword: TELESCOP
+      reffile:
+        title: Information about the reference file
+        type: object
+        properties:
+          type:
+            title: Reference file type
+            type: string
+            fits_keyword: REFTYPE
+          pedigree:
+            title: The pedigree of the reference file
+            type: string
+            fits_keyword: PEDIGREE
+          description:
+            title: Description of the reference file
+            type: string
+            fits_keyword: DESCRIP
+          author:
+            title: Author of the reference file
+            type: string
+            fits_keyword: AUTHOR
+          useafter:
+            title: Use after date of the reference file
+            type: string
+            fits_keyword: USEAFTER
+        required: [author, pedigree, description, useafter]
```

Comment:
Setting these to required to make sure reference file creators don't forget them. However, if current files in CRDS don't have them this may cause them to fail (not necessearily a bad thing).